sources/web/datalab/server.ts

```diff
@@ -303,8 +303,8 @@ export function run(settings: common.Settings): void {
   jupyter.init(settings);
   auth.init(settings);
   noCacheContent.init(settings);
-  reverseProxy.init();
-  sockets.init();
+  reverseProxy.init(settings.socketioPort);
```

Comment:
This should probably just be 'settings' instead of 'settings.socketioPort', since we'll soon want to add an 'ungitPort' value to the settings object too.